<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AUBus</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
QWidget#widget {
    background-color: qlineargradient(
        spread:pad, x1:0.1, y1:0.1, x2:0.9, y2:0.9,
        stop:0 rgba(10, 158, 190, 255),
        stop:1 rgba(170, 255, 255, 255)
    );
}
QPushButton {
    background-color: rgb(251, 201, 255);
    color: black;
    border-radius: 40px;
    border: 2px solid #cccccc;
    min-width: 200px;
    min-height: 60px;
    font: 63 12pt "Sitka Heading Semibold";
}
QPushButton:hover {
    background-color: #e6f7ff;
}
QPushButton:pressed {
    background-color: #b3ecff;
}
QLabel {
    color: black;
    font: 63 10pt "Sitka Heading Semibold";
}
QListWidget {
    background-color: white;
    border-radius: 20px;
    border: 2px solid #cccccc;
    padding: 10px;
    font: 10pt "Segoe UI";
    color: black;
}
QListWidget::item {
    border-bottom: 1px solid #eeeeee;
    padding: 15px;
}
QListWidget::item:selected {
    background-color: rgba(251, 201, 255, 100);
}
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacerTop">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="titleLabel">
      <property name="styleSheet">
       <string notr="true">
font: 63 36pt "Sitka Heading Semibold";
color: black;
       </string>
      </property>
      <property name="text">
       <string>RIDE HISTORY</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="subtitleLabel">
      <property name="styleSheet">
       <string notr="true">
font: 63 14pt "Sitka Heading Semibold";
color: black;
       </string>
      </property>
      <property name="text">
       <string>Your past AUBus journeys</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="statsLabel">
      <property name="styleSheet">
       <string notr="true">
background-color: rgba(0, 170, 255, 180);
border-radius: 15px;
font: 63 12pt "Sitka Heading Semibold";
color: black;
padding: 10px;
       </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="text">
       <string>üìä 24 Total Rides ‚Ä¢ ‚≠ê 4.8 Average Rating</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QListWidget" name="rideList">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="backButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>Back to Profile</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacerBottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
